mne/simulation/evoked.py

```diff
@@ -70,11 +70,14 @@ def simulate_evoked(fwd, stc, info, cov, nave=30, iir_filter=None,
     .. versionadded:: 0.10.0
     """
     evoked = apply_forward(fwd, stc, info, use_cps=use_cps)
+    if cov is None:
+        return evoked
+
     if nave < np.inf:
         noise = _simulate_noise_evoked(evoked, cov, iir_filter, random_state)
         evoked.data += noise.data / math.sqrt(nave)
         evoked.nave = np.int64(nave)
```

Comment:
This is incorrect as you now no longer set `nave` in the `cov=None` case

Reply:
... oh I guess this was always the case when `nave=np.inf`, but I'll fix it here

Reply:
argh I see now why you did it this way. I'm okay with it!